tests/conftest.py

```diff
@@ -2742,6 +2742,26 @@ def mock_config(rnafusion_dir: Path, rnafusion_case_id: str) -> None:
     ).touch(exist_ok=True)
 
 
+# Tomte fixtures
+
+
+@pytest.fixture(scope="function")
+def tomte_context(
```

Comment:
just a minor suggestion, could you add this fixture between the lines 274 and 478 of this file so that it is grouped with other context fixtures?

Reply:
I would like to keep tomte fixtures grouped as it is for rnafusion, taxproflier and so on

Reply:
Maybe we should think about moving all of these fixtures to a `nextflow` folder in `fixture_plugins`, but out of scope for this PR

Reply:
I do like this since it's quite messy currently. Let's do it in a future PR